Manual/Releases.lean

```diff
@@ -0,0 +1,26 @@
+/-
+Copyright (c) 2025 Lean FRO LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Joachim Breitner
+-/
+
+import VersoManual
+
+import Manual.Releases.«v4.15.0»
+import Manual.Releases.«v4.16.0»
+import Manual.Releases.«v4.17.0»
+
+open Manual
+open Verso.Genre
+
+#doc (Manual) "Release Notes" =>
+%%%
+tag := "release-notes"
+file := "releases"
+%%%
```

Comment:
```suggestion
number := false
%%%
```
I think this is more like an appendix or an index than it is like content in the manual, and this would be more clear if it were outside the numbering system.

I also found it a bit hard to focus on the section titles when they were next to the section numbers - both are sequences of digits and dots. I think this would solve that too.